vme/src/slog.h

```diff
@@ -0,0 +1,114 @@
+#pragma once
```

Comment:
For part 1 of the split PR, you want this file and with code removal from cpp file.